pkg/distro/detectv2.go

```diff
@@ -0,0 +1,86 @@
+package distro
+
+import (
+	"fmt"
+	"os"
+	"slices"
+
+	"github.com/go-git/go-git/v5"
+)
+
+// DetectV2 tries to detect which distro the user wants to operate on by seeing
+// if the current working directory is the distro's packages repo. No
+// advisory-related repositories are detected.
+func DetectV2() (Distro, error) {
```

Comment:
**Suggestion**: I don't have strong feelings about this functionality. I am just a bit concerned of the value of adding this detection versus explicitly requesting the distro to users.

Reply:
What do you mean by "explicitly requesting the distro to users"?

Detecting the distro this way was a suggestion from the delivery squad, and it's _less implicit_ than the `Detect()` function we already use elsewhere in the advisory commands, since it no longer detects your advisory repos.

Reply:
Good. I was missing that context 👍🏻 . That helped to alleviate my concerns 😃 .